okhttp/src/test/java/okhttp3/EventListenerTest.java

```diff
@@ -1118,7 +1118,7 @@ private void writeChunk(BufferedSink sink) throws IOException {
     assertThat(listener.recordedEventTypes()).containsExactly(
         "CallStart", "ProxySelectStart", "ProxySelectEnd", "DnsStart", "DnsEnd", "ConnectStart",
         "ConnectEnd", "ConnectionAcquired", "RequestHeadersStart", "RequestHeadersEnd",
-        "RequestBodyStart", "RequestFailed", "ConnectionReleased", "CallFailed");
+        "RequestBodyStart", "RequestFailed", "ResponseFailed", "ConnectionReleased", "CallFailed");
```

Comment:
Should we have a test for this where it does successfully read the response?

Reply:
Yep, the new test case includes that.